packages/shortcode/src/index.js

```diff
@@ -90,7 +90,7 @@ export function next( tag, text, index = 0 ) {
  * @return {string} Text with shortcodes replaced.
  */
 export function replace( tag, text, callback ) {
-	return text.replace( regexp( tag ), ( match, left, $3, attrs, slash, content, closing, right ) => {
+	return text.replace( regexp( tag ), function( match, left, $3, attrs, slash, content, closing, right ) {
```

Comment:
I don't know this package at all, but looking at `fromMatch` definition it's obvious that the code in `master` isn't correct:
https://github.com/WordPress/gutenberg/blob/3564bdb53718de5f02c754470be765037d216498/packages/shortcode/src/index.js#L223-L240